docs/connection.rst

```diff
@@ -16,13 +16,13 @@ Connection class is handy to have if you plan on building and executing
 queries in your application. It's more appropriate to store
 connection in a global variable or global class::
 
-    $app->db = atk4\dsql\Connection::connect($dsn, $user, $pass);
+    $app->db = atk4\dsql\Connection::establish($dsn, $user, $pass);
 
 
-.. php:staticmethod:: connect($dsn, $user = null, $password = null, $args = [])
+.. php:staticmethod:: establish($dsn, $user = null, $password = null, $args = [])
```

Comment:
one pitfall with passing password as an argument is that debugBacktrace could expose it. Could you think how to avoid the password being displayed if user have forgotten to turn off errors?